skaffold/src/test/kotlin/com/google/container/tools/skaffold/DefaultSkaffoldExecutorServiceTest.kt

```diff
@@ -226,4 +231,18 @@ class DefaultSkaffoldExecutorServiceTest {
 
         assertThat(result.commandLine).isEqualTo("skaffold dev --filename test.yaml")
     }
+
+    @Test
+    fun `isSkaffoldAvailable returns true when skaffold is available`() {
+        mockkStatic(System::class)
+        every { System.getenv("PATH") } answers { testSkaffoldFile.parent }
+        assertThat(defaultSkaffoldExecutorService.isSkaffoldAvailable()).isTrue()
+    }
+
+    @Test
+    fun `isSkaffoldAvailable returns false when skaffold is not available`() {
```

Comment:
It might also be a good idea to add a similar unit test, but with actual list of directories in mocked PATH, not only empty PATH, so we know for sure the parsing works.